include/deal.II/lac/la_vector.h

```diff
@@ -160,6 +160,15 @@ namespace LinearAlgebra
     reinit(const VectorSpaceVector<Number> &V,
            const bool omit_zeroing_entries = false) override;
 
+    /**
+     * Returns `false` as this is a serial block vector.
+     *
+     * This functionality only needs to be called if using MPI based vectors and
+     * exists in other objects for compatibility.
+     */
+    bool
+    has_ghost_elements() const;
```

Comment:
Is this from a different part of your work? It doesn't match the PR subject.

Reply:
No, we need that for being able to instantiate the function. It seems that this is the only vector class that is missing that function.